trial-config-service/src/test/resources/config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "trial": [
+    {
+      "id": "test_1",
+      "trialName":"Hello World Trial",
+      "trialId":"prefix/1",
```

Comment:
currently there is a bit of duplication. for eaxmple I am using similar json config in the mts trial deployment repo.
in any case, I think trialId should not have '/' as it might cause issues, e.g. If im using it for resources names...
how about prefix-1

Reply:
agree - will change

Reply:
We dont have trialId in the config.json file we are using from GIT so I will remove this trialId ref from this 'test' resource - it was probably copied from the initial config file from a few weeks ago..